pisek/config/config_types.py

```diff
@@ -28,6 +28,11 @@ class JudgeType(StrEnum):
     opendata_v1 = "opendata-v1"
 
 
+class FormatType(StrEnum):
```

Comment:
Maybe ``DataFormat`` would be better? ``FormatType`` doesn't say anything about what it describes.

Reply:
Sure. `DataFormat` is also pretty meaningless, but it would be consistent with the usage of "data" in pisek.

Reply:
Done.